com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2 extends java.lang.Object implements java.util.concurrent.Callable
final com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper this$0
final java.util.concurrent.Callable val$callable
    void <init>(com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper, java.util.concurrent.Callable)
    {
        com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2 r0;
        com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper $r1;
        java.util.concurrent.Callable $r2;

        r0 := @this: com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2;

        $r1 := @parameter0: com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper;

        $r2 := @parameter1: java.util.concurrent.Callable;

        r0.<com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2: com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper this$0> = $r1;

        r0.<com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2: java.util.concurrent.Callable val$callable> = $r2;

        specialinvoke r0.<java.lang.Object: void <init>()>();

        return;
    }
    public java.lang.Object call()
    {
        com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2 r0;
        java.util.concurrent.Callable $r1;
        java.lang.Object $r2;
        java.lang.Exception $r3;
        b.a.a.a.t $r4;

        r0 := @this: com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2;

     label1:
        $r1 = r0.<com.crashlytics.android.core.CrashlyticsExecutorServiceWrapper$2: java.util.concurrent.Callable val$callable>;

        $r2 = interfaceinvoke $r1.<java.util.concurrent.Callable: java.lang.Object call()>();

     label2:
        return $r2;

     label3:
        $r3 := @caughtexception;

        $r4 = staticinvoke <b.a.a.a.f: b.a.a.a.t h()>();

        interfaceinvoke $r4.<b.a.a.a.t: void e(java.lang.String,java.lang.String,java.lang.Throwable)>("CrashlyticsCore", "Failed to execute task.", $r3);

        return null;

        catch java.lang.Exception from label1 to label2 with label3;
    }
